tests/js/spec/components/organizations/timeRangeSelector/index.spec.jsx

```diff
@@ -143,6 +151,7 @@ describe('TimeRangeSelector', function() {
       relative: null,
       start: new Date('2017-10-10T04:00:00.000Z'),
       end: new Date('2017-10-18T03:59:59.000Z'),
+      utc: false,
```

Comment:
I'm a little confused here, the `Z` on the timestamp being used here indicates that it is in UTC timezone, but we then pass an extra `utc: false` flag to indicate that it's _not_ a UTC time and should be interpreted as local? I'm not sure if this means doing an actual timezone conversion between the UTC time passed and the local zone, or whether its just replacing the zone info with local time? 

I don't really have any context on this code, so feel free to dismiss this if I'm on the wrong track, but if [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date string already include the timezone information, we should never have to pass any additional state to indicate what zone we are talking about?

Reply:
So behavior is is that the initial state is the user wants to pick dates in UTC (midnight -> end of day) - and when they toggle UTC off, we want to maintain the current time of day of `00:00:00.000Z-04:00` (The tests run in EST) but we pass around dates with no timezone offset.

Reply:
So if I understand correctly `utc` is simply a display setting, and the actual date is given by the start and end values which do have a timezone?

Reply:
Yes it's a display setting. We also pass around a JS date object and not the ISO string, so we don't have any knowledge about UTC.